health-services/individual/src/main/java/org/egov/individual/service/IndividualService.java

```diff
@@ -415,4 +413,20 @@ private void integrateWithUserService(IndividualBulkRequest request,
             }
         }
     }
+    Boolean isSmsEnabledForRole(IndividualRequest request) {
+        List<String> roleCodes = new ArrayList<>();
+        if (StringUtils.isEmpty(properties.getSmsDisabledRoles()))
+            return true;
+        String[] smsDisabledRolesArray = properties.getSmsDisabledRoles().split(",");
```

Comment:
there is a way to load the list directly without splitting check in chat gpt or property code